src/shared/services/window/window.service.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { WindowService } from './window.service';
+
+describe('WindowService', () => {
+
+    let windowService: WindowService = new WindowService();
+    let userName = 'test user';
+
+    it('should be set local storage', ( ) => {
+        windowService.setLocalStorage('user', userName);
+
+        expect(windowService.getLocalStorage('user')).toBe('test user');
```

Comment:
Why use `'test user'` here when you've passed it `userName` above? It would be better to use the same string constant.

Reply:
Done